src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -113,10 +116,10 @@ export function ChangeDataView({
           }
         });
       }
-      setDataViewsList(dataViewsRefs);
+      setDataViewsList(hadnleAlphabeticalSorting(dataViewsRefs, orderDirection));
```

Comment:
not cleat to me why `orderDirection` is calculated out of that `useEffect`. If you did it for optimisation it's not working as you expected cause `orderDirection` will be executed on each render